apps/browser/qml/pages/components/HistoryContextMenu.qml

```diff
@@ -0,0 +1,45 @@
+/****************************************************************************
+**
+** Copyright (c) 2014 - 2021 Jolla Ltd.
+** Copyright (c) 2021 Open Mobile Platform LLC.
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.1
+import Sailfish.Silica 1.0
+
+Component {
+    ContextMenu {
+        MenuItem {
+            //% "Open in private tab"
+            text: qsTrId("sailfish_browser-me-open-in-private-tab")
+            onClicked: view.openInPrivateTab(model.url, model.title)
+        }
+        MenuItem {
+            //: Share link from browser history pulley menu
+            //% "Share"
+            text: qsTrId("sailfish_browser-me-share-link")
+            onClicked: webShareAction.shareLink(model.url, model.title)
+        }
+        MenuItem {
+            //% "Copy to clipboard"
+            text: qsTrId("sailfish_browser-me-copy-to-clipboard")
+            onClicked: Clipboard.text = model.url
```

Comment:
It would be good to add a simple notification to indicate the text has been copied. Something like [this](https://github.com/sailfishos/sailfish-browser/blob/fba7e57fd375e391d099f2b5179e52cc67440d35/apps/browser/qml/pages/LoginsPage.qml#L36).

Reply:
Done